meson.build

```diff
@@ -18,14 +18,22 @@ conf_data = configuration_data()
 conf_data.set('install_prefix', get_option('prefix'))
 conf_data.set('project_version', meson.project_version())
 conf_data.set('gettext_package', meson.project_name())
+conf_data.set('package_bindir', get_option('bindir'))
 conf_data.set('package_localedir', get_option('localedir'))
+conf_data.set('pkexec_path', get_option('pkexec_path'))
 
 configure_file(
   input: 'common/config-meson.h.in',
   output: 'config.h',
   configuration: conf_data
 )
 
+desktop_file = configure_file(
+  input: 'dists/showmethekey.desktop.in',
+  output: 'showmethekey.desktop',
+  configuration: conf_data
+)
```

Comment:
This can take an install_dir kwarg, rather than additionally specifying install_data down below.

Reply:
I don’t understand, what do you mean by install_dir kwarg?

Reply:
https://mesonbuild.com/Reference-manual_functions.html#configure_file

```meson
configure_file(
  input: 'dists/showmethekey.desktop.in',
  output: 'showmethekey.desktop',
  configuration: conf_data,
  install_dir: get_option('datadir') / 'applications',      # <<< This right here
)
```

Then you can delete this:
```meson

install_data(
  desktop_file,
  install_dir: get_option('prefix') / get_option('datadir') / 'applications'
)
```

Reply:
Ah, I see, thanks! Fixed in a5738388cb6851b33085dbc1765eb6896c03d145.

Reply:
Do we need `get_option('prefix')` in `configure_file`'s `install_dir`?

Reply:
No, because if datadir is just `share/` then Meson automatically prepends prefix to it for you.

Reply:
Maybe I should remove `get_option('prefix')` in other `install_dir` to make it looks clear.

Reply:
That's not a bad idea. :)